packages/react-composites/src/composites/ChatComposite/file-sharing/utils.ts

```diff
@@ -19,3 +21,12 @@ export const truncatedFileName = (fileName: string, length: number): string => {
  * @returns string
  */
 export const extension = (fileName: string): string => fileName.split('.').pop() || '';
+
+/**
+ * @private
+ *
+ * @param fileSharingMetadata
+ * @returns FileMetadata[]
+ */
+export const extractFileMetadata = (metadata: Record<string, string>): FileMetadata[] =>
+  JSON.parse(metadata.fileSharingMetadata);
```

Comment:
What happens here if there is no `fileSharingMetadata`, or if `metadata` is invalid JSON?

Reply:
added a check for fileSharingMetadata